data/summary/types/cgm_test.go

```diff
@@ -1488,10 +1482,14 @@ var _ = Describe("CGM Summary", func() {
 
 				for _, period := range periodKeys {
 					By(fmt.Sprintf("checking period %s", period))
-					Expect(userCGMSummary.Stats.Periods[period].StandardDeviation).To(BeNumerically("~", targetSD, 1.5))
+					// the range for this value is quite large, as the result is still RNG, we check the precise result in the final period
+					Expect(userCGMSummary.Stats.Periods[period].StandardDeviation).To(BeNumerically("~", targetSD, 4))
```

Comment:
We don't have anything that uses these intermediate values, do we?

Since they vary so much (which isn't that surprising), and if we don't use the intermediate values, it's probably ok not to test them.

Reply:
I'm mostly just checking that they are populated with something believable, but yes, since the generation hits 30d at once, 1d especially is likely to be very far off of the 30d value due to the sample size and rng.

I might make the generation return all 4 expected values, if i can find an elegant way to do calculate that without too much work, or ill just check that the values exist instead, as adding rng to tests passing is not funny for the guy who has to hit re-run 3 times on bad luck.